backends-common/rabbitmq/src/main/java/org/apache/james/backends/rabbitmq/ReactorRabbitMQChannelPool.java

```diff
@@ -106,11 +119,28 @@ public ReactorRabbitMQChannelPool(Mono<Connection> connectionMono, int poolSize)
         this.borrowedChannels = new ConcurrentSkipListSet<>(Comparator.comparingInt(System::identityHashCode));
     }
 
+    public void start() {
+        sender = createSender();
+    }
+
+    public Sender getSender() {
+        return sender;
+    }
+
+    public Receiver createReceiver() {
+        return RabbitFlux.createReceiver(new ReceiverOptions().connectionMono(connectionMono));
+    }
+
+    public Mono<Connection> getConnectionMono() {
+        return connectionMono;
+    }
+
     @Override
     public Mono<? extends Channel> getChannelMono() {
         return Mono.fromCallable(() -> {
             Channel channel = pool.borrowObject(MAXIMUM_BORROW_TIMEOUT_IN_MS);
             borrowedChannels.add(channel);
+            Preconditions.checkArgument(channel.isOpen());
```

Comment:
Maybe one line up

Reply:
Should be on https://github.com/linagora/james-project/pull/2741